python/tchannel/tornado/data.py

```diff
@@ -316,3 +316,22 @@ def close_argstreams(self, force=False):
         for stream in self.argstreams:
             if stream.auto_close or force:
                 stream.close()
+
+
+class ProtocolError(object):
+    """Object to represent protocol error message
+
+    Note: Internal use only.
```

Comment:
If this is only used in connection.py and is for internal use only there, declare it in that file.